ocs_ci/ocs/cluster_load.py

```diff
@@ -274,67 +296,100 @@ def get_query(self, query):
+            "iops": self.get_query(constants.IOPS_QUERY, mute_logs=mute_logs),
+            "used_space": self.get_query(constants.USED_SPACE_QUERY, mute_logs=mute_logs) / 1e+9
         }
 
-    def print_metrics(self):
+    def print_metrics(self, mute_logs=False):
         """
         Print metrics
 
+        Args:
+            mute_logs (bool): True for muting the Prometheus logs, False otherwise
+
         """
-        high_latency = 500
-        metrics = self.get_metrics()
-        limit_msg = ""
-        pods_msg = ""
-        if self.cluster_limit:
-            limit_msg = (
-                f"({metrics.get('iops') / self.cluster_limit * 100:.2f}% of the "
-                f"{self.cluster_limit:.1f} limit)\n"
-            )
-        if self.dc_objs:
-            pods_msg = (
-                f"\nNumber of pods running FIO: {len(self.dc_objs)}"
-            )
-        logger.info(
-            f"\n===============================\n"
-            f"Cluster throughput: {metrics.get('throughput'):.2f} MB/s\n"
-            f"Cluster latency: {metrics.get('latency'):.2f} ms\n"
-            f"Cluster IOPS: {metrics.get('iops'):.2f}\n{limit_msg}"
-            f"Cluster used space: {metrics.get('used_space'):.2f} GB{pods_msg}"
-            f"\n==============================="
+        high_latency = 200
+        metrics = self.get_metrics(mute_logs=mute_logs)
+        limit_msg = (
+            f" ({metrics.get('iops') / self.cluster_limit * 100:.2f}% of the "
+            f"{self.cluster_limit:.2f} limit)"
+        ) if self.cluster_limit else ""
+        pods_msg = f" || Number of FIO pods: {len(self.dc_objs)}" if self.dc_objs else ""
+        msg = (
+            f"Throughput: {metrics.get('throughput'):.2f} MB/s || "
+            f"Latency: {metrics.get('latency'):.2f} ms || "
+            f"IOPS: {metrics.get('iops'):.2f}{limit_msg} || "
+            f"Used Space: {metrics.get('used_space'):.2f} GB{pods_msg}"
         )
+        logger.info(f"Cluster utilization:{wrap_msg(msg)}")
         if metrics.get('latency') > high_latency:
             logger.warning(f"Cluster latency is higher than {high_latency} ms!")
+
+    def adjust_load_if_needed(self):
+        """
+        Dynamically adjust the IO load based on the cluster latency.
+        In case the latency goes beyond 400 ms, start deleting FIO pods.
+        Once latency drops back below 200 ms, re-create the FIO pods
+        to make sure that cluster load is around the target percentage
+
+        """
+        latency = self.calc_trim_metric_mean(
+            constants.LATENCY_QUERY, mute_logs=True
+        )
+        if latency > 0.4 and len(self.dc_objs) > 0:
+            msg = (
+                f"Latency is higher than 400 ms ({latency * 1000:.2f} ms)."
+                f" Lowering IO load by deleting an FIO pod that is running"
+                f" in the test background. Once the latency drops back to "
+                f"less than 200 ms, FIO pod will be re-spawned"
```

Comment:
you are not exposing any variable here so you should omit the `f` here

Reply:
The first line of the message has a variable. Are you sure it is ok?

Reply:
NVM, fixed